Content.Shared/Hands/Components/SharedHandsComponent.cs

```diff
@@ -892,6 +892,7 @@ public class Hand
         [ViewVariables]
         public IContainer? Container { get; set; }
 
+        // TODO: Make this a nullable EntityUid...
```

Comment:
easy: merge #5634 .... oh.. more conflicts...

Reply:
this is so sad

Reply:
Once this is merged I'll fix conflicts for that and the verb one, and then try bully people into merging them before more appear.